src/components/Search.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+
+class Search extends React.Component {
+    constructor() {
+        super();
+
+        this.state= {
```

Comment:
It looks like `query` stored both in Search and App. I'd say if you store query in App, then it would make sense to pass it down to Search as props.